lib/store/settings_store.dart

```diff
@@ -486,6 +486,11 @@ abstract class SettingsStoreBase with Store {
   Node getCurrentNode(WalletType walletType) {
     final node = nodes[walletType];
 
+    // TODO: What is this for.
+    if (walletType == WalletType.decred) {
+      return Node();
+    }
```

Comment:
My preliminary research seems to suggest that it allows a user choose a custom address to connect to. The user's preferred node would be saved to the `nodes` map and we shouldn't need to do this if check.

Reply:
Ohhh.

Reply:
Changed the todo do `Implement connecting to a user's preferred node.`